core/controllers/base.py

```diff
@@ -390,7 +390,6 @@ def validate_and_normalize_args(self):
             else:
                 request_arg_keys.append(arg)
                 handler_args[arg] = self.request.get(arg)
-
```

Comment:
For this line, I wasn't able to pass the lint checks until I added this space

Reply:
This seems odd, especially since no other changes are being made to this file. Could you please explain which lint check(s) failed?